frontend/src/components/housing/submission/SubmissionForm.vue

```diff
@@ -399,6 +402,19 @@ onMounted(async () => {
         label="Contact email"
         :disabled="!props.editable"
       />
+      <Dropdown
+        class="col-3"
+        name="feedbackConsent"
+        label="Research opt-in"
+        :placeholder="props.submission.feedbackConsent ? 'Yes' : 'No'"
+        :disabled="!props.editable"
+        :options="['true', 'false']"
+        :option-label="
+          (e: string) => {
+            return e === 'true' ? BasicResponse.YES : BasicResponse.NO;
+          }
+        "
+      />
```

Comment:
Could we not use the BasicResponse values as the options, and have line 203 check `submitData.feedbackConsent = values.feedbackConsent === BasicResponse.YES;` ?